vault/resource_identity_group.go

```diff
@@ -18,13 +18,15 @@ func identityGroupResource() *schema.Resource {
 		Read:   identityGroupRead,
 		Delete: identityGroupDelete,
 		Exists: identityGroupExists,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
 
 		Schema: map[string]*schema.Schema{
 			"name": {
 				Type:        schema.TypeString,
-				Required:    true,
+				Optional:    true,
```

Comment:
Ditto for `identity_entity` above.

Reply:
Vault should auto-generate names so it might be optional as well.